arch/arm/src/phy62xx/irq.c

```diff
@@ -37,6 +37,14 @@
 #include "arm_internal.h"
 #include "jump_function.h"
 #include "bus_dev.h"
+  #define __PACKED                               __attribute__((packed))
+
+__PACKED typedef struct {
+    uint16_t opcode;        /**< operation code, defined mesh profile 3.7.3.1 Operation codes */
+    uint16_t company_id;    /**< SIG ID: 0xFFFF */
+} mible_mesh_opcode_t;
+
```

Comment:
Why place here?

Reply:
i will check this part with my leader.
then decide preserve it or not.